Makefile

```diff
@@ -70,6 +70,7 @@ internal/feature/flags.go: internal/feature/flags.yml
 	$(GO_GENERATE) ./internal/feature
 
 libflux: $(LIBFLUX_GENERATED_TARGETS)
+	cd libflux && $(CARGO) check # this checks for missing docs
```

Comment:
I think this would be better as part of the test-rust target instead of the build. That way you can still build the code while working to get docs/tests passing. Thoughts?

Reply:
Makes sense!